ui/src/timeMachine/actions/queryBuilder.ts

```diff
@@ -140,6 +144,21 @@ export const setAggregateWindow = (period: string) => ({
   payload: {period},
 })
 
+export const setAggregateFillValues = (fillValues: boolean) => ({
+  type: 'SET_AGGREGATE_FILL_VALUES' as 'SET_AGGREGATE_FILL_VALUES',
+  payload: {fillValues},
+})
+
+export const setIsAutoWindowPeriod = (isAutoWindowPeriod: boolean) => ({
+  type: 'SET_IS_AUTO_WINDOW_PERIOD' as 'SET_IS_AUTO_WINDOW_PERIOD',
+  payload: {isAutoWindowPeriod},
+})
+
+export const setIsAutoFunction = (isAutoFunction: boolean) => ({
+  type: 'SET_IS_AUTO_FUNCTION' as 'SET_IS_AUTO_FUNCTION',
+  payload: {isAutoFunction},
+})
+
```

Comment:
i know the rest of these follow the old pattern but...

```suggestion
export const setAggregateFillValues = (fillValues: boolean) => ({
  type: 'SET_AGGREGATE_FILL_VALUES',
  payload: {fillValues},
} as const)
```
etc..